src/query/expression/tests/it/schema.rs

```diff
@@ -166,8 +167,7 @@ fn test_schema_from_struct() -> Result<()> {
     let schema = create_test_complex_schema();
     let flat_column_ids = schema.to_leaf_column_ids();
 
-    let (leaf_column_ids, leaf_fields) = schema.leaf_fields();
-    assert_eq!(leaf_column_ids, schema.to_leaf_column_ids());
```

Comment:
if remove `assert_eq!(leaf_column_ids, schema.to_leaf_column_ids());`, MUST add other test that merge leaf_fields.column_ids() to a vector equal to `schema.to_leaf_column_ids()`